app/components/marketplace/stripe_overview_component.rb

```diff
@@ -0,0 +1,12 @@
+# Renders a tile with a link to view the Marketplace Stripe account page
+class Marketplace::StripeOverviewComponent < ApplicationComponent
+  attr_accessor :marketplace
+  attr_accessor :marketplace_stripe_utility
```

Comment:
It's slightly more ruby-esque to do `delegate :stripe_utility, to: :marketplace, prefix: true`.

Reply:
Oh, nice tip!